app/api/suggestions/suggestions.ts

```diff
@@ -228,19 +136,19 @@ export const Suggestions = {
 
   saveMultiple: async (_suggestions: IXSuggestionType[]) => {
     const toSave: IXSuggestionType[] = [];
-    const update: IXSuggestionType[] = [];
+    let doStateUpdate = false;
```

Comment:
Let's talk about this, I don't completely see the reason.

Reply:
Will rework this after discussion